dev-docs/howtoguides/testing.md

```diff
@@ -38,10 +38,17 @@ By default, integration tests will do the following on a given cloud platform:
  * Install the appropriate ubuntu-advantage-tools and ubuntu-advantage-pro deb
  * Run the integration tests on that instance.
 
-The testing can be overridden to run using a local copy of the ubuntu-advantage-client source code instead of the daily PPA by providing the following environment variable to the behave test runner:
-```UACLIENT_BEHAVE_BUILD_PR=1```
+The testing can be overridden to run installing ubuntu-advantage-tools from other places instead of the daily PPA by providing `UACLIENT_BEHAVE_INSTALL_FROM` to the behave test runner. The default is `UACLIENT_BEHAVE_INSTALL_FROM=daily`, and the other available options are:
```

Comment:
I think we can change to `The testing can be overridden to install ubuntu-advantage-tools from...`

Reply:
done